config.yaml

```diff
@@ -143,3 +143,13 @@ options:
       disable TX checksumming on the vxlan.calico network interface. This works
       around an upstream issue in Calico:
       https://github.com/projectcalico/calico/issues/3145
+  service-cluster-ips:
```

Comment:
Consider making these config options match what's in the specification (linked in mattermost). If they don't match, then when the cluster is upgraded back to mainline, the existing config will be dropped.

Reply:
Fixed.

Reply:
Sorry, wrong file was committed. Fixing that.

Reply:
Now fixed.